lib/engine.py

```diff
@@ -132,6 +133,7 @@ def source_gen():
                                    for n in range(len(linevalues))]
                 except Exception as e:
                     self.warning('Exception in line %s: %s' % (self.table.record_id, e))
+                    exit()
```

Comment:
Is this going to cause issues for `try_install_all.py` and/or for testing? I'm not familiar enough with exit to know for sure, but I think it might just completely end the Python process if it's triggered. I'm also not sure what it gets us that simply throwing the exception doesn't.